examples/grafana-alloy-auto-instrumentation/ebpf-otel/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:22.04 as builder
+RUN apt-get update && apt-get -y install wget gcc
+RUN wget https://go.dev/dl/go1.22.10.linux-amd64.tar.gz
+RUN tar -C /usr/local -xzf go1.22.10.linux-amd64.tar.gz
+# fixed versions for pyroscope, otel-collector, otel-profiler due to protocol changes
+RUN wget https://github.com/open-telemetry/opentelemetry-ebpf-profiler/archive/9eb454541a3cf1282a8755dbcec9a9471b243e83.tar.gz
```

Comment:
I like this. However, I think we should update to avoid working with pinned versions and images once the profiler is stable enough